### ISSUE TITLE
Text Format Toolbar

### ISSUE THREAD

**Issue by hammannja** [JackAdams/editable-text-demo#1]
Hi Jack,

I was able to get your demonstration application up and running in Meteor, but could not get the Text Format Toolbar, the Save button or the "New Post" button to show.

Is this functionality and these images part of the demo? If so, I would like to use them. It is a key feature that attracted me to your code.

Thanks!

Jerry

**Comment by JackAdams** [comment]
Cloning this repo and running the app should produce the same app with all the buttons and things in place.

If you can give me a meteorpad.com sample of your code, I'll be able to help troubleshoot.

**Comment by hammannja** [comment]
Sorry, GitHub cloning was something I could not master (yet), even after downloading the GitHub desktop app. I will need to learn this.

I did however, find the solution. I needed to add more packages than initially disclosed in the quick starts:
https://atmospherejs.com/babrahams/editable-text and
https://atmospherejs.com/babrahams/editable-list 

I needed to add these packages:
babrahams:editable-list
babrahams:editable-text-wysiwyg
twbs:bootstrap-noglyph
fortawesome:fontawesome
babrahams:editable-text-wysiwyg-bootstrap-3
meteorhacks:fast-render (this one may not have been necessary)

Everything seems to be working fine now! Thanks for this useful code! It appears to be exactly what I am looking for!

Jerry
